tests/functional/commands/test_db_load.py

```diff
@@ -0,0 +1,462 @@
+STEP 2 Check benchmarks 0 3
+STEP 2 Check benchmarks 1 3
+STEP 2 Check benchmarks 2 3
+STEP 2 Check benchmarks 3 3
+STEP 3 Check destination experiments 0 1
+STEP 3 Check destination experiments 1 1
+STEP 4 Check source experiments 0 3
+STEP 4 Check source experiments 1 3
+STEP 4 Check source experiments 2 3
+STEP 4 Check source experiments 3 3
+STEP 5 Delete data to replace in destination 0 0
+STEP 6 Insert new data in destination 0 6
+STEP 6 Insert new data in destination 1 6
+STEP 6 Insert new data in destination 2 6
+STEP 6 Insert new data in destination 3 6
+STEP 6 Insert new data in destination 4 6
+STEP 6 Insert new data in destination 5 6
+STEP 6 Insert new data in destination 6 6
+STEP 1 Collect source experiments to load 0 1
+STEP 1 Collect source experiments to load 1 1
+STEP 2 Check benchmarks 0 3
+STEP 2 Check benchmarks 1 3
+STEP 2 Check benchmarks 2 3
+STEP 2 Check benchmarks 3 3
+STEP 3 Check destination experiments 0 1
+STEP 3 Check destination experiments 1 1
+STEP 4 Check source experiments 0 3
+STEP 4 Check source experiments 1 3
+STEP 4 Check source experiments 2 3
+STEP 4 Check source experiments 3 3
+STEP 5 Delete data to replace in destination 0 12
+STEP 5 Delete data to replace in destination 1 12
+STEP 5 Delete data to replace in destination 2 12
+STEP 5 Delete data to replace in destination 3 12
+STEP 5 Delete data to replace in destination 4 12
+STEP 5 Delete data to replace in destination 5 12
+STEP 5 Delete data to replace in destination 6 12
+STEP 5 Delete data to replace in destination 7 12
+STEP 5 Delete data to replace in destination 8 12
+STEP 5 Delete data to replace in destination 9 12
+STEP 5 Delete data to replace in destination 10 12
+STEP 5 Delete data to replace in destination 11 12
+STEP 5 Delete data to replace in destination 12 12
+STEP 6 Insert new data in destination 0 6
+STEP 6 Insert new data in destination 1 6
+STEP 6 Insert new data in destination 2 6
+STEP 6 Insert new data in destination 3 6
+STEP 6 Insert new data in destination 4 6
+STEP 6 Insert new data in destination 5 6
+STEP 6 Insert new data in destination 6 6
+""".strip()
+    )
+
+
+def test_load_bump_no_benchmarks(empty_database):
+    """Test load all database with --resolve --bump"""
+    data_source = LOAD_DATA
+
+    storage = setup_storage()
+    loaded_db = storage._db
+    assert len(loaded_db.read("benchmarks")) == 0
+    assert len(loaded_db.read("experiments")) == 0
+    assert len(loaded_db.read("trials")) == 0
+    assert len(loaded_db.read("algo")) == 0
+
+    execute(f"db load {data_source}")
+    benchmarks = loaded_db.read("benchmarks")
+    experiments = loaded_db.read("experiments")
+    trials = loaded_db.read("trials")
+    algos = loaded_db.read("algo")
+    assert len(benchmarks) == 0
+    assert len(experiments) == 3
+    assert len(trials) == 24
+    assert len(algos) == 3
+
+    execute(f"db load {data_source} -r bump")
+    # Duplicated data should be bumped, so we must expect twice quantity of data.
```

Comment:
Here it is particularly important that we test for the links since we changed the base references (experiment id).